test/CMakeLists.txt

```diff
@@ -45,23 +45,29 @@ endforeach()
 ## Test-specific data, properties and dependencies
 set_target_properties(logging PROPERTIES COMPILE_DEFINITIONS LOGGING_LEVEL_ALL)
 
+add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/test/data/tz.sqlite
+  COMMAND ${CMAKE_COMMAND} -E make_directory test/data/
+  COMMAND ${VALHALLA_SOURCE_DIR}/scripts/valhalla_build_timezones > ${CMAKE_BINARY_DIR}/test/data/tz.sqlite
+  COMMENT "Building tz.sqlite..."
+  WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
+)
+
 add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/test/data/utrecht_tiles/0/003/196.gph
   COMMAND ${CMAKE_COMMAND} -E make_directory test/data/utrecht_tiles/
-  COMMAND ${VALHALLA_SOURCE_DIR}/scripts/valhalla_build_timezones 2>/dev/null > test/data/utrecht_tiles/tz.sqlite
   COMMAND ${CMAKE_BINARY_DIR}/valhalla_build_tiles
-      --inline-config '{"mjolnir":{"tile_dir":"test/data/utrecht_tiles","timezone":"test/data/utrecht_tiles/tz.sqlite","hierarchy":true,"shortcuts":true,"concurrency":4,"logging":{"type":""}}}'
+      --inline-config '{"mjolnir":{"tile_dir":"test/data/utrecht_tiles","timezone":"test/data/tz.sqlite","hierarchy":true,"shortcuts":true,"concurrency":1,"logging":{"type":""}}}'
       ${VALHALLA_SOURCE_DIR}/test/data/utrecht_netherlands.osm.pbf
   COMMAND ${CMAKE_BINARY_DIR}/valhalla_add_predicted_traffic
-      --inline-config '{"mjolnir":{"tile_dir":"test/data/utrecht_tiles","concurrency":4,"logging":{"type":""}}}'
+      --inline-config '{"mjolnir":{"tile_dir":"test/data/utrecht_tiles","concurrency":1,"logging":{"type":""}}}'
```

Comment:
Was setting the concurrency to 1 needed to get the tests to build?

Reply:
Hmm This was a leftover from trying out all the various things.

That said, we are now building 5 different tile-sets on a full test-run and there are more pbf's being added in a different PR. 5 tilesets with 4 threads each results in 20 running threads.

It may just be best to do this single threaded.